src/js/background.js

```diff
@@ -1,7 +1,18 @@
-chrome.browserAction.onClicked.addListener((tab) => {
-	chrome.tabs.executeScript(tab.id, {
+chrome.browserAction.onClicked.addListener(tab => executeScript(tab.id));
+
+chrome.commands.onCommand.addListener(command => {
+	if (command === "convert_currency") {
+		chrome.tabs.query({
+			active: true,
+			currentWindow: true
+		}, tabs => executeScript(tabs[0].id));
+	}
+});
+
+function executeScript(id) {
```

Comment:
Can move this function to the top.

Reply:
Hoisting?

Reply:
Just to follow conventions. (Declaration is before usage)